python/ray/tune/execution/trial_runner.py

```diff
@@ -198,6 +198,7 @@ def _serialize_and_write():
             exclude = ["*/checkpoint_*"]
 
         if self._syncer:
+            # Todo: Implement sync_timeout
```

Comment:
can you finish this comment please? it reads a bit confusing because we are implementing sync_timeout with this PR :)

Reply:
Ok, updated!